src/work/financial-wellness-app/index.md

```diff
@@ -6,7 +6,16 @@ cover: metlife-home.png
 date: 2019-03-09
 ---
 
-Sorry, I'm still working on this case study. 
+An insurance company in the US wanted to find a way to help their clients’ employees to better understand their finances and improve their decision-making. The product needed to provide employers important information about their employees’ financial wellness, and give them the ability to recommend insurance products based upon user needs.
+
+As part of a team of designers and behavioural scientists, I researched, designed and iterated upon the core proposition that aimed to generate behavioural change and habit building in users. This required rounds of remote testing to investigate a number of approaches to forming good financial habits for users, and to get to the core of what would make users want to return to the app. We also ran co-creation sessions with the client to build out something that would solve both business and user problems.
+
+![](https://cdn-images-1.medium.com/max/2000/1*UlKhjaBjWPm8vY7w8K93QQ.png)
+
+## **Results**
```

Comment:
```suggestion
## Results
```

Not super important but you probs don't need to make headings `<strong>`, since headings are already rendered with the bold font variant